AdminUi/apps/admin_ui/lib/main.dart

```diff
@@ -63,17 +67,23 @@ final _router = GoRouter(
   ],
 );
 
-class MainApp extends StatelessWidget {
-  const MainApp({super.key});
+class AdminUIApp extends StatelessWidget {
```

Comment:
I think the casing in this project is AdminUi

Reply:
Ah sorry didn't read it correctly.